gemdaq-testing/gemhardware/src/common/amc13/AMC13Manager.cc

```diff
@@ -168,6 +170,52 @@ void gem::hw::amc13::AMC13Manager::initializeAction()
   p_amc13->AMCInputEnable(m_slotMask);
   usleep(500);
 
+  // Use local TTC signal if config doc says so
+  p_amc13->localTtcSignalEnable(m_enableLocalTTC);
+
+  // Enable Monitor Buffer Backpressure if config doc says so
+  p_amc13->monBufBackPressEnable(m_monBackPressEnable);
+
+  // m_dtc->configurePrescale(1,m_preScaleFactNumOfZeros);
+  p_amc13->configurePrescale(0, m_prescaleFactor);
+
+  // set the FED id
+  p_amc13->setFEDid(m_fedID);
+
+  // reset the T1
+  p_amc13->reset(::amc13::AMC13::T1);
+
+  // reset the T1 counters
+  p_amc13->resetCounters();
+
+  // Setting L1A if config doc says so
+  // How to configure all parameters for L1A trigger? configureLocalL1A(m_enableLocalL1A,mode,burst,rate,rules) -> refer to http://cactus.web.cern.ch/cactus/nightly/api/html_dev_amc13/d5/df6/classamc13_1_1_a_m_c13.html#a800843bf6119f6aaeb470e406778a9b2
+  //p_amc13->configureLocalL1A(m_enableLocalL1A,0,1,m_internalPeriodicPeriod,0);
+  p_amc13->configureLocalL1A(m_enableLocalL1A,2,1,1000,0);
+
+  // setup the monitoring helper -> TO BE DEVELOPED!!!
```

Comment:
From here down to line 217 (in this version) we don't really do anything with this, so it probably isn't necessary.
We shouldn't just blindly copy the implementation as it is in HCAL because our use case may be different, so I'd focus on the bare minimum functionality that we need for now, and then worry about adding in functionality as necessary, so as to not clutter up the code too much.